subjects/devops/json-researcher/README.md

```diff
@@ -0,0 +1,60 @@
+## json-researcher
+
+### Instructions
+
+Create the script `json-researcher.sh` wich will show the "name" and the "power" of the superhero with `id: 1`
+
+- Where to look : https://((DOMAIN))/assets/superhero/all.json
+
+- What to use : `curl` and `jq` and `grep`
+
+- The output should be exactly like the example bellow but with the expected name
+
+```console
+$./json-researcher.sh | cat -e
+  "name": "<...>",$
+    "power": <...>,$
+$
+```
+
+### Hints
+
+With `curl` you can get the content of the file from the url:
+```console
+$curl https://assets.01-edu.org/devops-branch/HeadTail.txt
+<...>
+$
+```
+
+`jq` is a command-line utility that can slice, filter, and transform the components of a JSON file
+
+```console
+$cat fruit.json
+{
+  "name": "apple",
+  "color": "green",
+  "price": 1.0
+}
+$jq '.color' fruit.json
+"green"
+$
+```
+
+`grep` is a command-line utility for searching plain-text data sets for lines that match a regular expression.
+
+```console
+$cat fruit.json  
+{
+  "name": "apple",
+  "color": "green",
+  "price": 1.0
+}
+$cat fruit.json | grep color
+  "color": "green",
+$
+```
+
+May you will need to use `pipe (|)` and `&&`.
```

Comment:
I will rephrase this as follows:
You will need to use pipes (`|`) and the logical operator `&&`

Reply:
they are different solutions to solve this exercise, and it's just a hint

Reply:
fixed in all exercises